care/apps/patients/serializers.py

```diff
@@ -111,7 +122,22 @@ def create(self, validated_data):
         patient_diseases = validated_data.pop("patient_diseases", None)
         patient = super().create(validated_data)
         if patient_facility:
-            patient_models.PatientFacility.objects.create(**patient_facility, patient=patient)
+            transfer_facility = patient_facility.pop("transfer_facility", None)
+            from_facility = patient_models.PatientFacility.objects.create(**patient_facility, patient=patient)
+            if transfer_facility:
+                admitted_at = patient_facility.pop("discharged_at", None)
+                to_facility = patient_models.PatientFacility.objects.create(
+                    facility=transfer_facility,
+                    admitted_at=admitted_at,
+                    patient_status_id=patient_constants.ADMITTED_TO_FACILITY_STATUS_VALUE,
+                    patient=patient,
+                )
+                patient_models.PatientTransfer.objects.create(
+                    from_patient_facility=from_facility,
+                    to_facility=transfer_facility,
+                    status=patient_constants.TRANSFER_STATUS.ACCEPTED,
+                    status_updated_at=datetime.now(),
+                )
```

Comment:
No need to add another key transfer_facility. In warse-case users may add 3-4 facilities. 

Make "patient_facility" as a list of dict and each dict will represent a PatientFacililty.